configs/experimental/yolo_town1_obstacles.conf

```diff
@@ -0,0 +1,56 @@
+###### Object detection config ######
+--obstacle_detection
+--nosimulator_obstacle_detection
+# --obstacle_detection_model_paths=dependencies/models/obstacle_detection/faster-rcnn/
+# --obstacle_detection_model_names=faster-rcnn
+# --path_coco_labels=dependencies/models/pylot.names
+# --path_model_outputs=dependencies/models/pylot.outputs
+--obstacle_detection_model_paths=dependencies/models/obstacle_detection/yolo/
+--obstacle_detection_model_names=yolo
+--path_coco_labels=dependencies/models/carla_obstacles.names
+--path_model_outputs=dependencies/models/custom.outputs
+--path_yolo_anchors=dependencies/models/yolo3_anchors.txt
+--obstacle_detection_gpu_memory_fraction=0.2
+--obstacle_detection_min_score_threshold=0.5
+#--evaluate_obstacle_detection
+###### Depth #####
+--perfect_depth_estimation
+###### Segmentation config ######
+--perfect_segmentation
+###### Prediction config ######
+--obstacle_tracking
+--tracking_num_steps=10
+--prediction
+--prediction_type=linear
+--prediction_num_past_steps=10
+--prediction_num_future_steps=10
+###### Planning config #####
+--planning_type=waypoint
+###### Control config #####
+--control=mpc
+#--lane_detection
+#--perfect_lane_detection
+--stop_for_vehicles=True
+--stop_for_people=True
+--stop_for_traffic_lights=False
+#--target_speed=40
+###### Carla config ######
+--random_seed=618549084
```

Comment:
This is the seed used to reproduce the scenario where the ego car crashes into a vehicle that's stopped at a red light.

Reply:
Could you please add a brief comment to this line that summarizes this test scenario?

Reply:
Done